tags.json

```diff
@@ -234,6 +275,11 @@
     "paint",
     "color"
   ],
+  "bug": [
```

Comment:
`insect`?

Reply:
https://en.wikipedia.org/wiki/Insect

Reply:
https://en.wikipedia.org/wiki/Bug?wprov=sfla1

> More broadly, any of many small, primarily terrestrial invertebrates, such as Insects

Reply:
yeah true, but it's just an tag to help people search for icons. 
So any related words or part of something or synonyms are welcome.